### DISCUSSION TITLE
Correct Origin position after "lonlat to sphere"

### DISCUSSION THREAD

**Discussion by dennart** [domlysz/BlenderGIS#476] [General]
Hello there!

Faced with the fact that after importing shp-files and transforming into a spherical object, the origin of the objects remains in those places where the object was originally imported. Is it possible to adjust the function (procedure) in such a way that the origin of coordinates also gets a new correct place together with the object.

See examples screeshot below.
[Import SHP](https://dennart.ru/share_images/blendergis_origin_01.jpg)
[lonlat to sphere](https://dennart.ru/share_images/blendergis_origin_02.jpg)
[Set Origin command](https://dennart.ru/share_images/blendergis_origin_03.jpg)
[Incorrect Origin after Transform](https://dennart.ru/share_images/blendergis_origin_04.jpg)
[Correct place of Origin](https://dennart.ru/share_images/blendergis_origin_05.jpg)

I will be very happy if the feature is implemented in the next version.

Beforehand thanx.

**Comment by domlysz** [comment]
Issue #484
